packages/client/hmi-client/src/components/Sidebar.vue

```diff
@@ -24,75 +37,71 @@ const enum Mode {
 	Profile = 'Profile'
 }
 
-const auth = useAuthStore();
-
-// Get sidebar position if saved in local storage
-const isSidebarPositionRight = ref(
-	localStorage.getItem('isSidebarPositionRight')
-		? localStorage.getItem('isSidebarPositionRight') === 'true'
-		: false
-);
-const selectedMode = ref('');
-const isCollapsed = computed(() => selectedMode.value.length === 0);
+const selectedView = ref('');
 
-// Later move mode specific features into their own components
-const logout = () => {
-	auth.logout();
-	window.location.assign('/logout');
-};
-
-function updateMode(mode: string) {
-	selectedMode.value = mode === selectedMode.value ? '' : mode;
-}
-
-function moveSidebar() {
-	localStorage.setItem('isSidebarPositionRight', (!isSidebarPositionRight.value).toString());
-	isSidebarPositionRight.value = localStorage.getItem('isSidebarPositionRight') === 'true';
+/**
+ * Open a View
+ * @param {string} view - The view to be open.
+ * @param {boolean} [openViewSidePanel=true] - Should the side-panel be open when opening the view.
+ */
+function openView(view: string, openViewSidePanel: boolean = true): void {
+	selectedView.value = view;
+	if (isSidePanelOpen.value) {
+		if (!openViewSidePanel) closeSidePanel();
+	} else if (openViewSidePanel) openSidePanel();
+
+	if (view === View.SimulationPlan) {
+		router.push({ name: 'simulation' });
```

Comment:
can we just use the value of "name" to be the equivalent enum value instead of hard-coding the "simulation" string

Reply:
No worries. Not a priority anyway!